appinfo/app.php

```diff
@@ -22,9 +21,7 @@
  *
  */
 
-namespace OCA\Files3d\AppInfo;
```

Comment:
Namespaces are linked to folders, this was not the proper folder here :)

Reply:
I remember such a problem from another project...thanks!